pydis_site/apps/content/README.md

```diff
@@ -0,0 +1,37 @@
+# The "content" app
+
+This application serves static, markdown-based content. Django-wise there is
+relatively little code in there, most of it is concerned with serving our
```

Comment:
This is a comma splice.
```suggestion
relatively little code in there; most of it is concerned with serving our
```

Reply:
ca2c583b15d46d7821a03ec047678429f96c7e03